services/brig/src/Brig/User/API/Auth.hs

```diff
@@ -90,7 +90,8 @@ routesPublic = do
     Doc.notes
       "This operation generates and sends a login code. \
```

Comment:
Did you check that this is only for phone login?  Then the docs should say so and make the distinction between this and `/verification-code/send` more clear.

```suggestion
      "This operation generates and sends a login code via sms for phone login. \
```